tests/suites/test_suite_aria.function

```diff
@@ -0,0 +1,318 @@
+/* BEGIN_HEADER */
+#include "mbedtls/aria.h"
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * depends_on:MBEDTLS_ARIA_C
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE */
+void aria_encrypt_ecb( char *hex_key_string, char *hex_src_string,
+                           char *hex_dst_string, int setkey_result )
+{
+    unsigned char key_str[1000];
```

Comment:
Why 1000? Add a comment explaining the rationale behind selecting this number, and make a constant if that helps readability. Someone who later attempts to adapt this test to run on a device that doesn't have 4 KiB of stack may want to know why they might not be able to reduce the sizes used here.

Also, consider using heap or globals for these buffers instead of stack for better portability for on-target testing.